### PR TITLE
update request and delete request

### PR DESCRIPTION
**Pull request by manickaa**
Update request endpoint: [https://idvmpyv72b.execute-api.us-east-1.amazonaws.com/dev/requestEditByShelter/{shelterOwnerName}/{customerName}/{shelterName}/{animalName}](https://idvmpyv72b.execute-api.us-east-1.amazonaws.com/dev/requestEditByShelter/{shelterOwnerName}/{customerName}/{shelterName}/{animalName})

Table before:
Screen Shot 2022-02-27 at 10 05 36 PM

Editing request for user 'aishtest7':
Screen Shot 2022-02-27 at 10 05 53 PM

Table After:
Changed the request status to 'adopted' for 'aishtest7' and cancelled other user's (aishtest2 and aishtest6) requests to the same pet
Screen Shot 2022-02-27 at 10 06 27 PM
Changes the availability of the pet (snowy4_petco2) to 'adopted'
Screen Shot 2022-02-27 at 10 06 42 PM

Delete request endpoint: [https://idvmpyv72b.execute-api.us-east-1.amazonaws.com/dev/requestDelete/{userName}/{shelterName}/{animalName}](https://idvmpyv72b.execute-api.us-east-1.amazonaws.com/dev/requestDelete/{userName}/{shelterName}/{animalName})

Screen Shot 2022-02-27 at 10 07 16 PM

Table after deleting the request:
Screen Shot 2022-02-27 at 10 16 43 PM

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -7,7 +7,6 @@
         "@types/axios": "^0.14.0",
         "@types/http-errors": "^1.6.1",
         "aws-api-gateway-client": "^0.3.7",
-        "aws-xray-sdk": "^2.2.0",
```
Comment (manickaa) [inline backend/package.json:10]: Have removed xray since it was taking too much space and slowing the deployment

File: backend/webpack.config.js [hunk 2]
```diff
@@ -3,10 +3,10 @@ const slsw = require('serverless-webpack');
 // var nodeExternals = require('webpack-node-externals')
 
 module.exports = {
-  mode: slsw.lib.webpack.isLocal ? 'development' : 'production',
+  mode: 'production',
```
Comment (manickaa) [inline backend/webpack.config.js:6]: Changes to speed deployment.

File: backend/webpack.config.js [hunk 3]
```diff
@@ -3,10 +3,10 @@ const slsw = require('serverless-webpack');
...
   entry: slsw.lib.entries,
   // externals: [nodeExternals()],
-  devtool: 'source-map',
+  //devtool: 'source-map',
```
Comment (manickaa) [inline backend/webpack.config.js:9]: Did not need source map. Was taking too much space

**Review by kruegead — APPROVED** [review]
This all looks good to me

**Pull request merged**